src/tuf/target.ts

```diff
@@ -43,11 +49,18 @@ async function getTargetPath(tuf: Updater, target: string): Promise<string> {
   try {
     targetInfo = await tuf.refresh().then(() => tuf.getTargetInfo(target));
   } catch (err) {
-    throw new InternalError(`error refreshing TUF metadata: ${err}`);
+    throw new InternalError({
+      code: 'TUF_METADATA_REFRESH_ERROR',
+      message: 'error refreshing TUF metadata',
+      cause: err,
+    });
   }
 
   if (!targetInfo) {
-    throw new InternalError(`target ${target} not found`);
+    throw new InternalError({
+      code: 'TUF_TARGET_NOT_FOUND',
```

Comment:
maybe a bit funky to fit the pattern?
```suggestion
      code: 'TUF_FIND_TARGET_ERROR',
```